sessionize/lib/src/commonTest/kotlin/co/touchlab/sessionize/EventModelTest.kt

```diff
@@ -35,9 +37,9 @@ class EventModelTest {
         val eventModel = EventModel("67316")
         val session = AppContext.sessionQueries.sessionById("67316").executeAsOne()
         val si = collectSessionInfo(session)
-        /*eventModel.toggleRsvpSuspend(si)
+        eventModel.toggleRsvpSuspend(si)
         assertTrue { sessionizeApiMock.rsvpCalled }
-        assertTrue { analyticsApiMock.logCalled }*/
+        assertTrue { analyticsApiMock.logCalled }
```

Comment:
Should we add an assertion that the notification gets called too?

Reply:
Sure, I can add that.